pkg/machines/components/vmSnapshotsTab.css

```diff
@@ -0,0 +1,12 @@
+.no-snapshots {
+    text-align: center;
+    line-height: 3.0rem
+}
+
+.snap-greyed-out {
+    color: #72767b;
```

Comment:
I think we might have a color variable for this. Maybe `--pf-global--palette--black-600`?
https://www.patternfly.org/v4/documentation/overview/global-css-variables#global-css-variables